docs/designers-developers/developers/block-api/block-transforms.md

```diff
@@ -39,6 +39,7 @@ A transformation of type `block` is an object that takes the following parameter
 - **blocks** _(array)_: a list of known block types. It also accepts the wildcard value (`"*"`), meaning that the transform is available to _all_ block types (eg: all blocks can transform into `core/group`).
 - **transform** _(function)_: a callback that receives the attributes and inner blocks of the block being processed. It should return a block object or an array of block objects.
 - **isMatch** _(function, optional)_: a callback that receives the block attributes and should return a boolean. Returning `false` from this function will prevent the transform from being available and displayed as an option to the user.
+- **isMultiblock** _(boolean, optional)_: whether the transformation can be applied when multiple blocks are selected. If this is true, the `transform` function will receive as paramaters an with the values of each selected block. False by default.
```

Comment:
You missed the word `array` in `will receive as paramaters an with...` and maybe replace `values` with `attributes`?

Also the API returns a second argument that is an array with the `InnerBlocks` of each selected block.

https://github.com/WordPress/gutenberg/blob/c1208188447b78e2b4e983c8b8f21a9f6c0d27f9/packages/blocks/src/api/factory.js#L463

Reply:
Good catch! Pushed a new wording. Is now clearer? Will welcome suggestions. Today, my brain has proven to have issues to enter _documentation mode_.